src/bp-activity/classes/class-bp-rest-activity-link-preview-endpoint.php

```diff
@@ -98,7 +98,7 @@ public function get_items( $request ) {
 		if ( empty( $parse_url_data ) ) {
 			return new WP_Error(
 				'bp_rest_unknown_error',
-				__( 'Sorry! preview is not available right now. Please try again later.', 'buddyboss' ),
+				__( 'There was a problem generating a link preview.', 'buddyboss' ),
```

Comment:
@jitendrabanjara1991 Please move this change into API repo and create PR for the same.